Lib/test/test_userlist.py

```diff
@@ -11,6 +11,16 @@ class UserListTest(list_tests.CommonTest):
     def test_init(self):
         super().test_init()
 
+    # TODO: RUSTPYTHON
+    @unittest.expectedFailure
+    def test_contains_fake(self):
+        super().test_contains_fake()
+
+    # TODO: RUSTPYTHON
+    @unittest.expectedFailure
+    def test_count(self):
+        super().test_count()
```

Comment:
When I add things like this, I'd like to direct contributors against leaving the last two lines of this pattern behind when they un-mark these tests.